lib/bing_ads_ruby_sdk/railtie.rb

```diff
@@ -0,0 +1,13 @@
+require "bing_ads_ruby_sdk"
+require "rails"
+
+module BingAdsRubySdk
+  class Railtie < Rails::Railtie
+    railtie_name :bing_ads_ruby_sdk
+
+    rake_tasks do
+      path = File.expand_path(__dir__)
+      Dir.glob("#{path}/tasks/**/*.rake").each { |f| load f }
+    end
+  end
+end
```

Comment:
@danielmbarlow Why didn't we need that before ? is it related to a Rails upgrade?

Reply:
I think we did need it before and we didn't have it. I tried running the rake task from within my Rails project and it wasn't found.

This file includes the rake task in whichever app includes the gem.